magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -64,8 +64,8 @@ MAGIC:
 
     muon_ring:
         thr_low: 25
-        tailcut: [8, 4]
```

Comment:
was the change of the thresholds intended? 12,8 looks quite large, did it come from your optimization?

Reply:
It is not optimised, but it followed a comment (by Abelardo?) that if we take only very bright pixels the biases from NSB should be reduced even if the charge extractor is not optimal.